CONTRIBUTORS.md

```diff
@@ -10,24 +10,36 @@ Additionally, the PR needs to be approved by one of the core project members
 
 ## Build
 
-To obtain sources and build the `ci-operator` binary, use `go get` and `go build`:
+To obtain sources and build the binaries provided by this repo, run the following commands:
 
 ```
 $ go get github.com/openshift/ci-tools
 $ cd ${GOPATH}/src/github.com/openshift/ci-tools
-$ go build ./cmd/ci-operator
+$ make install
```

Comment:
well, we are using `go build ./cmd/ci-operator` to actually build the images in the parent directory in order to use them in the `Dockerfiles`. So `make bin-build` or just `make build` makes more sense here.